extensions/amp-story/1.0/amp-story-share-menu.js

```diff
@@ -131,6 +131,21 @@ export class ShareMenu {
     this.shareWidget_.loadRequiredExtensions(getAmpdoc(this.parentEl_));
     this.element_ = getAmpSocialSystemShareTemplate(this.parentEl_);
 
+    const shareText =
+      this.storeService_.get('title') +
+      ' ' +
+      getLocalizationService(this.parentEl_).getLocalizedString(
+        LocalizedStringId.AMP_STORY_SHARE_BY
+      ) +
+      ' ' +
+      this.parentEl_.getAttribute('publisher') +
```

Comment:
I'd recommend not doing this. Likely negatively impacts CTR. The preview generated from the URL does the same job.

Reply:
Agree that the preview right now is already generating a thumbnail with title and domain on popular platforms like Facebook and Twitter. Part of the original purpose of it is to give more information via text in case there are platforms doesn't have good url extraction(but I did double check various social platforms and only the native ones like SMS, email, and Telegram doesn't do url extraction).

I guess the question now becomes whether more information via text is better for CTA or not, for all platforms. Do we have currently have a way to first do a LE to see the CTA before fully launched this? Or as Malte said, this is an already known fact that longer text(or the mention of #WebStories) means worse CTA? @gmajoulet @newmuis

Reply:
We are not changing the card. The goal is to enhance the text with more details to encourage CTR.

Reply:
@cramforce do you still think we should not be adding this sharing text on top of the generated preview? Please let us know if so, so we can update our feature trackers. Thanks!

Reply:
Not up to me to decide, but generally I think users find this annoying. High CTR posts require the author to write a custom note, the author info is typically redundant with the generated card that are available in basically all social media apps, and it is unclear how the hashtag adds value to the end user (and it likely reduces the CTR).